py/specter/test/test_psf.py

```diff
@@ -486,23 +486,23 @@ def test_xyrange(self):
         self.assertGreater(xmax, 0)
 
 #- Test Pixellated PSF format
-class TestPixPSF(TestPSF):
+class TestPixPSF(GenericPSFTests,unittest.TestCase):
```

Comment:
I think this is fine, but what is the motivation for making this multiple inheritance instead of a linear inheritance chain?  i.e. originally TestPixPSF -> TestPSF -> unittest.TestCase vs. now TestPixPSF -> GenericPSFTests+unittest.TestCase